### PR TITLE
Add template for creating a webhook function with transform capabilities

### PR DESCRIPTION
**Pull request by rarayudu**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
* 
*
*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@rarayudu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (rarayudu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 201-logic-app-transform-function/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "existingFunctionAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the function container in which the function has to be created."
+            }
+        },
+        "functionName": {
```
Comment (singhkay) [inline 201-logic-app-transform-function/azuredeploy.json]: is this existing function or a new one?

File: 201-logic-app-transform-function/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
...reserved.\r\n// </copyright>\r\n//-----------------------------------------------------------------------\r\n\r\nusing System;\r\nusing System.Net;\r\n\r\n[System.Diagnostics.CodeAnalysis.SuppressMessage(\"Microsoft.Design\", \"CA1032:ImplementStandardExceptionConstructors\", Justification = \"Not needed.\")]\r\n[System.Diagnostics.CodeAnalysis.SuppressMessage(\"Microsoft.Usage\", \"CA2237:MarkISerializableTypesWithSerializable\", Justification = \"Not needed.\")]\r\npublic class IntegrationXsltException : Exception\r\n{\r\n    public IntegrationXsltException()\r\n    {\r\n    }\r\n\r\n    public IntegrationXsltException(string message) : base(message, null)\r\n    {\r\n        this.StatusCode = HttpStatusCode.InternalServerError;\r\n    }\r\n\r\n    public IntegrationXsltException(HttpStatusCode statusCode, string message)\r\n        : base(message)\r\n    {\r\n        this.StatusCode = statusCode;\r\n    }\r\n\r\n    public IntegrationXsltException(HttpStatusCode statusCode, string message, Exception innerException)\r\n        : base(message, innerException)\r\n    {\r\n        this.StatusCode = statusCode;\r\n    }\r\n\r\n    public HttpStatusCode StatusCode { get; set; }\r\n}",
```
Comment (singhkay) [inline 201-logic-app-transform-function/azuredeploy.json]: This is pretty weird. What is all this text?

File: 201-logic-app-transform-function/azuredeploy.json (hunk 1 above)
Comment (rarayudu) [inline 201-logic-app-transform-function/azuredeploy.json]: With this template we create a new function (for transform capabilities) in the existing function app/function container.

This template expects a already existing function app as input. This is the reason CI is also failing. I verified the templates manually.

File: 201-logic-app-transform-function/azuredeploy.json (hunk 2 above)
Comment (rarayudu) [inline 201-logic-app-transform-function/azuredeploy.json]: Azure functions provide a way to execute small pieces of code on the cloud in a sandbox environment.

The purpose of this template is to create one such function with XSLT trasnfrom capabilities for logic apps integration scenarios.

In this template we are specifying the files and their code and that gets executed..

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**